### ISSUE TITLE
CIFAR-100 dataset split

### ISSUE THREAD

**Issue by hellcodes** [SHI-Labs/Compact-Transformers#39]
Hello. First, thanks for sharing your impressive work.

Since I want to train CCT models for CIFAR-100 for reproduction,
I'd like to ask you about how to split the dataset for train/val/test.

Thanks.

**Comment by alihassanijr** [comment]
Hi,
Thank you for your interest.
We used the PyTorch version of the datasets, so we didn't evaluate on a validation split. We just trained on the training set and evaluated on the test set.
The new training script, which is a fork of `timm`'s, supports CIFAR-10/100.
We will be releasing new checkpoints on all datasets presented in the paper, along with their full configuration file soon.

If you have any more questions, feel free to reopen the issue.
